packages/generator-rest/src/request-builder/api-request-builder-class.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  ClassDeclarationStructure,
+  MethodDeclarationStructure,
+  OptionalKind,
+  ParameterDeclarationStructure,
+  StructureKind
+} from 'ts-morph';
+import { toPascalCase, toPropertyFormat } from '@sap-cloud-sdk/core';
+import { OpenApiPath, OpenApiServiceMetadata } from '../open-api-types';
+
+export function apiRequestBuilderClass(
+  serviceMetadata: OpenApiServiceMetadata
+): ClassDeclarationStructure {
+  return {
+    kind: StructureKind.Class,
+    name: `${serviceMetadata.apiName}ApiRequestBuilder`,
+    methods: method(serviceMetadata.paths),
+    isExported: true
+  };
+}
+
+function method(paths: OpenApiPath[]): MethodDeclarationStructure[] {
+  return paths.flatMap(p => getMethods(p));
```

Comment:
perhaps you can find adjust the spec file that the sample service also contains more than one method  (`calculateViaRestWithTenant`)

Reply:
I'll add more spec files for testing the generator.
I would also keep this service as e2e test.

Reply:
sure